pkg/ddevapp/templates.go

```diff
@@ -7,10 +7,9 @@ const ConfigInstructions = `
 # name: <projectname> # Name of the project, automatically provides
 #   http://projectname.ddev.site and https://projectname.ddev.site
 
-# type: <projecttype>  # backdrop, cakephp, craftcms, drupal, drupal6, drupal7, laravel, magento, magento2, php, shopware6, silverstripe, typo3, wordpress
+# type: <projecttype>  # backdrop, cakephp, craftcms, drupal6, drupal7, drupal8, drupal9, drupal10, drupal11, laravel, magento, magento2, php, shopware6, silverstripe, typo3, wordpress
```

Comment:
I added `drupal` back, because it was in the docs for the project type.
Perhaps a note about an alias (just like in the docs) is also needed here.

```suggestion
# type: <projecttype>  # backdrop, cakephp, craftcms, drupal, drupal6, drupal7, drupal8, drupal9, drupal10, drupal11, laravel, magento, magento2, php, shopware6, silverstripe, typo3, wordpress
```

Reply:
Thanks. I always wonder if this whole comment section is useful. Wonder if anybody reads it.

Reply:
i am using and reading it every time i dont spin up a project with a copy and paste snippet or an ansible script. i prefer using `ddev config` then. 

@stasadev and in regard of re-adding `drupal`. that is one point i've asked randy to remove before on the PR. for me the main problem in that list is that it is long and drupal alone makes up  roughly one third of the entries with eight different project types. and drupal and drupal11 are redundant since drupal is an alias for drupal11 as the current major version. my thought was enable users that know and use the alias "drupal" simply use it but for the visible options go with the absolute explicit project types of drupal6-drupal11 only. that was my thought behind suggesting removing drupal in that list. but as i told already to randy probably the better choice for the flow used on `ddev config` instead of going with such a lengthy inline list might be for example adding a fourth step. where the user is able for CMSs with more than one project type to select the specific version there. ( but that is out of the scope for this PR)

Reply:
Yeah, I also don't like that we have too many drupal project types.

Probably a fourth step in the interactive `ddev config` would be better, for example:

```
Project Type [backdrop, cakephp, craftcms, drupal, laravel, magento, magento2, php, shopware6, silverstripe, typo3, wordpress] (php): drupal

Note: The project type 'drupal' is aliased 'drupal11' in the current DDEV.
Project Version [drupal, drupal6, drupal7, drupal8, drupal9, drupal10, drupal11] (drupal):
```